### PR TITLE
classname added to svg attributes

### PR DESCRIPTION
**Pull request by Gautam2010**
#548 
Adding feature to add class name in layer option.

### CODE REVIEW

**Comment by Gautam2010** [comment]
> @Gautam2010 please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement

`@microsoft-github-policy-service agree [company="self"]`

**Comment by danmarshall** [comment]
Thanks for this PR, very much appreciated!

File: packages/maker.js/src/core/svg.ts [hunk 1]
```diff
@@ -412,7 +413,8 @@ namespace MakerJs.exporter {
             fillRule: "evenodd",
             fontSize: '9pt',
             useSvgPathOnly: true,
-            viewBox: true
+            viewBox: true,
+            className:""
```
Comment (danmarshall) [inline packages/maker.js/src/core/svg.ts:417]: I think this is not needed, as line 420 will merge it if it exists.

File: packages/maker.js/test/svg.js [hunk 2]
```diff
@@ -101,7 +101,7 @@ describe('Export SVG', function () {
         const svg = makerjs.exporter.toSVG(model, exportOptions);
         const expected = [
             '<svg width="50" height="50" viewBox="0 0 50 50" xmlns="http://www.w3.org/2000/svg">',
-                '<g id="svgGroup" stroke-linecap="round" fill-rule="evenodd" font-size="8pt" stroke="#333" stroke-width="0.5mm" fill="green" style="stroke:#333;stroke-width:0.5mm;fill:green">',
+                '<g id="svgGroup" stroke-linecap="round" fill-rule="evenodd" font-size="8pt" stroke="#333" stroke-width="0.5mm" fill="green" style="stroke:#333;stroke-width:0.5mm;fill:green" class="">',
```
Comment (danmarshall) [inline packages/maker.js/test/svg.js:104]: Let's add `className="test"` to `exportOptions` and then put `class="test"` here.

**Review by danmarshall — CHANGES_REQUESTED** [review]
Looking good! Just a couple of small changes.

File: packages/maker.js/src/core/svg.ts (hunk 1 above)
Comment (Gautam2010) [reply packages/maker.js/src/core/svg.ts]: Removed `className `from here

File: packages/maker.js/test/svg.js (hunk 2 above)
Comment (Gautam2010) [reply packages/maker.js/test/svg.js]: Added and tested with `className="test"`

**Review by danmarshall — APPROVED** [review]

**Comment by danmarshall** [comment]
Looks like the CLA bot is hung up. @Gautam2010 can you take a look at the reply format here:
https://github.com/microsoft/ContributorLicenseAgreement#accepting

**Comment by Gautam2010** [comment]
> Looks like the CLA bot is hung up. @Gautam2010 can you take a look at the reply format here: https://github.com/microsoft/ContributorLicenseAgreement#accepting

`@microsoft-github-policy-service agree`

**Comment by Gautam2010** [comment]
> I think you might need to just reply with only `@microsoft-github-policy-service agree` ?

@danmarshall I am wondering I just have to copy here in comment or branch? I have already added in comment but still workflow says it pending .

**Comment by danmarshall** [comment]
I think just copy here in a comment

**Comment by Gautam2010** [comment]
> > Looks like the CLA bot is hung up. @Gautam2010 can you take a look at the reply format here: https://github.com/microsoft/ContributorLicenseAgreement#accepting
> 
> `@microsoft-github-policy-service agree`

@danmarshall I did here

**Comment by danmarshall** [comment]
I'm not sure, but I think the bot wants only the `@microsoft-github-policy-service agree` and not any other text such as the blockquote.

**Comment by Gautam2010** [comment]
@microsoft-github-policy-service agree

**Comment by Gautam2010** [comment]
@danmarshall Many thanks in helping me my first PR

**Pull request merged**